server/bazelrunner/src/main/kotlin/org/jetbrains/bsp/bazel/bazelrunner/BazelRunner.kt

```diff
@@ -130,7 +130,11 @@ class BazelRunner(
   ) {
     val envString = processEnv?.let { envToString(it) }
     val directoryString = directory?.let { "cd $it &&" }
-    val processArgsString = processArgs.joinToString("' '", "'", "'")
+    val processArgsString = if (processArgs.size > 50) {
```

Comment:
Should we make this truncation configurable? Some folks may use these commands (however long) for debugging.

Reply:
That makes sense. It may turn out not to a big issue and to be a welcome change when we open a PR upstream. I'd let the upstream maintainers decide.